VideoLocker/src/main/java/org/edx/mobile/module/prefs/UserPrefs.java

```diff
@@ -43,6 +46,14 @@ public File getDownloadFolder() {
         File packDir = new File(downloadsDir, context.getPackageName());
         File edxDir = new File(packDir, profile.username);
         edxDir.mkdirs();
+        try {
+            File noMediaFile = new File(edxDir, ".nomedia");
+            if (!noMediaFile.createNewFile()) {
+                logger.debug(".NoMedia file not created");
+            }
```

Comment:
That's not necessary. According to the docs, ``createNewFile`` will just return false if the file already exists, which suggests to me that the log line in that case isn't necessary, since that's expected behavior